include/engine/api/route_api.hpp

```diff
@@ -228,42 +228,50 @@ class RouteAPI : public BaseAPI
 
         std::vector<util::json::Object> annotations;
 
-        if (parameters.annotations_type != RouteParameters::AnnotationsType::None)
+        if (parameters.annotations_type != RouteParameters::AnnotationsType::None ||
+            parameters.annotations == true)
         {
+            auto ReqAnnotations = parameters.annotations_type;
```

Comment:
local variables use `lower_case` names. Maybe `annotations_type` would be a good name here:

```
auto annotations_type = parameters.annotations_type;
```